opendatakit/aggregate
# Issue 371 remove missing odk tables pref

Closes #371

(Ignoring whitespaces in the diff settings is recommended while reviewing)

#### What has been done to verify that this works as intended?
Manually tested:
- The preference is no longer shown in the Site Admin > Preferences tab
- Launching publish and export tasks works as expected. No errors in the logs

#### Why is this the best possible solution? Were any other approaches considered?
This is a straightforward change. Almost all the changes were made using safe refactoring techniques.

#### Are there any risks to merging this code? If so, what are they?
Nope.

#### Do we need any specific form for testing your changes? If so, please attach one
No.

#### Does this change require updates to documentation? If so, please file an issue at https://github.com/opendatakit/docs/issues/new and include the link below.
Yes, along with the rest of v2.0 changes
